### PR TITLE
Change Operator provider to CNCF

### PR DESCRIPTION
**Pull request by objectiser**
Change provider from `Jaeger` to `CNCF` to enable it to be listed with other CNCF project operators.

@awgreene Would you be able to review and confirm this an appropriate change to update the provider used on operatorhub.io? If so, how does the update get applied to the hub?

Signed-off-by: Gary Brown <gary@brownuk.com>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/263)

**Comment by jpkrohling** [comment]
Just waiting for @awgreene's OK to merge this.

**Comment by objectiser** [comment]
@awgreene Is it ok to change the provider? and if so, is this PR ok or is anything else required?

**Comment by awgreene** [comment]
If CNFF is the provider please feel free to update the field.

Best practice is to create a new csv vand add the `replaces` field. Just make sure to create a PR against the [community-oeprators repo](https://github.com/operator-framework/community-operators/tree/master/community-operators)

**Comment by objectiser** [comment]
@jpkrohling I've updated the release.sh based on @awgreene 's comment about specifying the `replaces` field - so that this should automatically now be set to the version it is replacing. Would be good if we can get this in before doing the release.

File: .travis/release.sh [hunk 1]
```diff
@@ -22,6 +22,10 @@ sed "s/currentCSV: jaeger-operator.*/currentCSV: jaeger-operator.v${OPERATOR_VER
 sed "s~containerImage: docker.io/jaegertracing/jaeger-operator.*~containerImage: docker.io/${BUILD_IMAGE}~gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
 sed "s/name: jaeger-operator\.v.*/name: jaeger-operator.v${OPERATOR_VERSION}/gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
 sed "s~image: jaegertracing/jaeger-operator.*~image: ${BUILD_IMAGE}~gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
+
+export PREVIOUS_OPERATOR_VERSION=`grep "version: [0-9]" deploy/olm-catalog/jaeger-operator.csv.yaml | cut -f4 -d' '`
+sed "s/replaces: jaeger-operator\.v.*/replaces: jaeger-operator.v${PREVIOUS_OPERATOR_VERSION}/gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
```
Comment (jpkrohling) [inline .travis/release.sh]: Shouldn't this be cumulative? Like, 1.11.0 replaces 1.10.0, 1.9.0, ... ?

Comment (objectiser) [reply .travis/release.sh]: I think it is just a chaining mechanism - I've only looked at a couple of examples but they only listed the version they directly replaced.
@awgreene Is that correct?

Comment (awgreene) [reply .travis/release.sh]: It is a chaining mechanism.

**Pull request merged**